CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## Next Release
+
+- Adds new billing functionality for ReferralCustomer users
```

Comment:
```suggestion
- Adds new beta billing functionality for ReferralCustomer users
```

Nit: I would indicate this is `beta` in the changelog

Reply:
Will add that when we release